alphadia/workflow/peptidecentric/requantification_handler.py

```diff
@@ -0,0 +1,348 @@
+import pandas as pd
+from alphabase.peptide.fragment import get_charged_frag_types
+from alphabase.spectral_library.base import SpecLibBase
+from alphabase.spectral_library.flat import SpecLibFlat
+
+from alphadia.data.alpharaw_wrapper import AlphaRaw
+from alphadia.data.bruker import TimsTOFTranspose
+from alphadia.fragcomp.utils import add_frag_start_stop_idx, candidate_hash
+from alphadia.plexscoring.config import CandidateConfig
+from alphadia.plexscoring.plexscoring import CandidateScoring
+from alphadia.plexscoring.utils import (
+    candidate_features_to_candidates,
+    multiplex_candidates,
+)
+from alphadia.reporting.reporting import Pipeline
+from alphadia.workflow.config import Config
+from alphadia.workflow.managers.calibration_manager import CalibrationManager
+from alphadia.workflow.managers.fdr_manager import FDRManager
+
+
+class RequantificationHandler:
```

Comment:
I think we shouldn't put them in one handler. The naming is similar but it's actually two different things not really connected.

One is needed for transfer learning the other is needed for multiplexing.